tests/test_http_ext_auth.py

```diff
@@ -152,10 +152,27 @@ def __exit__(self, *exc):
 class TestHttpExtAuth(tb.ExtAuthTestCase):
     TRANSACTION_ISOLATION = False
 
-    EXTENSION_CONFIG = [
-        ("ext::auth::AuthConfig::auth_signing_key", f"<str>'{'a' * 32}'"),
-        ("ext::auth::AuthConfig::github_client_secret", f"<str>'{'b' * 32}'"),
-        ("ext::auth::AuthConfig::github_client_id", f"<str>'{uuid.uuid4()}'"),
+    EXTENSION_SETUP = [
+        f"""
+        CONFIGURE CURRENT DATABASE SET
+        ext::auth::AuthConfig::auth_signing_key := <str>'{'a' * 32}';
+        """,
+        (
+            """
+            CONFIGURE CURRENT DATABASE
+            INSERT ext::auth::ClientConfig {
```

Comment:
No need to split the docstring like that, curly braces can be quoted with another curly brace: `{{` and `}}`